source/sass/global/fonts.scss

```
@@ -0,0 +1,29 @@
+@font-face {
+  font-family: "oswald";
```

Обычно все-таки имена шрифтов пишут с заглавной буквы
```suggestion
  font-family: "Oswald";
```

---

Исправила